modules/server_processes_manager/php/mriuploadserverprocess.class.inc

```diff
@@ -105,6 +105,18 @@ class MriUploadServerProcess extends AbstractServerProcess
             throw $exception;
         }
 
+        // Check that paths exist before using.
+        if (!file_exists($this->_mriCodePath)) {
+            throw new \LorisException(
+                "Source code for imaging upload could not be found!"
+            );
+        }
+        if (!is_dir($this->_mriCodePath)) {
+            throw new \ConfigurationException(
+                "Value for MRI code path is not a directory!"
```

Comment:
Why is one of these a ConfigurationException but the other is a LorisException? In fact, I'm not sure why the file_exists one is even necessary. If it doesn't exist it's not a directory and the message is still valid.

Reply:
Sure, I'll remove the file check.

Reply:
Yea I was about to say file_exists checks for directory as well, but I'm not knowledgeable enough to know what we want to expect for `$this->_mriCodePath`, whether we want it as a directory or we allow a file..

Reply:
@Jkat Yeah I think the file_exists check was unnecessary and didn't make sense. I think I left it in from an earlier phase of the fix. `$this->_mriCodePath` should be the parent folder of the perl script that does image uploading.